pkgs/development/python-modules/graphite-web/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy3k, which
+, django, django_1_8
+, django_tagging, django_tagging_0_4_3
+, whisper, pycairo, cairocffi, ldap, memcached, pytz, urllib3, scandir
+}:
+if django         != django_1_8
+|| django_tagging != django_tagging_0_4_3
+then throw "graphite-web should be build with django_1_8 and django_tagging_0_4_3"
+else buildPythonPackage rec {
+  pname = "graphite-web";
+  version = "1.1.3";
+
+  disabled = isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0nvyq1859abdch2l90avsjk9vb03s7wgxgrjsqvjhf2b9c1ncsfh";
+  };
+
+  propagatedBuildInputs = [
+    django django_tagging whisper pycairo cairocffi
+    ldap memcached pytz urllib3 scandir
+  ];
+
+  postInstall = ''
+    wrapProgram $out/bin/run-graphite-devel-server.py \
+      --prefix PATH : ${which}/bin
```

Comment:
You can use `makeWrapperArgs` instead.

Reply:
I tried the following:
```
makeWrapperArgs = [ "--prefix PATH : ${which}/bin" ];
```
but with that the test suite fails to start up graphite-web:

```
one# [    7.566444] graphiteWeb-pre-start[706]: Traceback (most recent call last):
one# [    7.568265] graphiteWeb-pre-start[706]:   File "/nix/store/k8xndnycd95iqs0ky9nlcvqqnyx9p63v-python2.7-Django-1.8.18/bin/..django-admin.py-wrapped-wrapped", line 7, in <module>
one# [    7.574263] graphiteWeb-pre-start[706]:     management.execute_from_command_line()
one# [    7.582220] graphiteWeb-pre-start[706]:   File "/nix/store/h5pfcf6j3rl85xm4apggqg1d4za4yj53-python-2.7.15-env/lib/python2.7/site-packages/django/core/management/__init__.py", line 354, in execute_from_command_line
one# [    7.595708] graphiteWeb-pre-start[706]:     utility.execute()
one# [    7.602375] graphiteWeb-pre-start[706]:   File "/nix/store/h5pfcf6j3rl85xm4apggqg1d4za4yj53-python-2.7.15-env/lib/python2.7/site-packages/django/core/management/__init__.py", line 328, in execute
one# [    7.611485] graphiteWeb-pre-start[706]:     django.setup()
one# [    7.617331] graphiteWeb-pre-start[706]:   File "/nix/store/h5pfcf6j3rl85xm4apggqg1d4za4yj53-python-2.7.15-env/lib/python2.7/site-packages/django/__init__.py", line 18, in setup
one# [    7.626695] graphiteWeb-pre-start[706]:     apps.populate(settings.INSTALLED_APPS)
one# [    7.635331] graphiteWeb-pre-start[706]:   File "/nix/store/h5pfcf6j3rl85xm4apggqg1d4za4yj53-python-2.7.15-env/lib/python2.7/site-packages/django/apps/registry.py", line 85, in populate
one# [    7.646523] graphiteWeb-pre-start[706]:     app_config = AppConfig.create(entry)
one# [    7.652897] graphiteWeb-pre-start[706]:   File "/nix/store/h5pfcf6j3rl85xm4apggqg1d4za4yj53-python-2.7.15-env/lib/python2.7/site-packages/django/apps/config.py", line 86, in create
one# [    7.665890] graphiteWeb-pre-start[706]:     module = import_module(entry)
one# [    7.674238] graphiteWeb-pre-start[706]:   File "/nix/store/7sbvn0wgv7hsnxpss1jba723kx5nz6d4-python-2.7.15/lib/python2.7/importlib/__init__.py", line 37, in import_module
one# [    7.682407] graphiteWeb-pre-start[706]:     __import__(name)
one# [    7.693954] graphiteWeb-pre-start[706]:   File "/nix/store/h5pfcf6j3rl85xm4apggqg1d4za4yj53-python-2.7.15-env/lib/python2.7/site-packages/opt/graphite/webapp/graphite/functions/__init__.py", line 10, in <module>
one# [    7.703974] graphiteWeb-pre-start[706]:     from graphite.logger import log
one# [    7.710845] graphiteWeb-pre-start[706]:   File "/nix/store/h5pfcf6j3rl85xm4apggqg1d4za4yj53-python-2.7.15-env/lib/python2.7/site-packages/opt/graphite/webapp/graphite/logger.py", line 106, in <module>
one# [    7.720270] graphiteWeb-pre-start[706]:     log = GraphiteLogger() # import-shared logger instance
one# [    7.725289] graphiteWeb-pre-start[706]:   File "/nix/store/h5pfcf6j3rl85xm4apggqg1d4za4yj53-python-2.7.15-env/lib/python2.7/site-packages/opt/graphite/webapp/graphite/logger.py", line 39, in __init__
one# [    7.732729] graphiteWeb-pre-start[706]:     level = logging.DEBUG if settings.DEBUG else logging.INFO,
one# [    7.736779] graphiteWeb-pre-start[706]:   File "/nix/store/h5pfcf6j3rl85xm4apggqg1d4za4yj53-python-2.7.15-env/lib/python2.7/site-packages/opt/graphite/webapp/graphite/logger.py", line 78, in _config_logger
one# [    7.743253] graphiteWeb-pre-start[706]:     handler = Rotater(log_file, when=when, backupCount=backupCount)
one# [    7.748248] graphiteWeb-pre-start[706]:   File "/nix/store/7sbvn0wgv7hsnxpss1jba723kx5nz6d4-python-2.7.15/lib/python2.7/logging/handlers.py", line 171, in __init__
one# [    7.754796] graphiteWeb-pre-start[706]:     BaseRotatingHandler.__init__(self, filename, 'a', encoding, delay)
one# [    7.762192] graphiteWeb-pre-start[706]:   File "/nix/store/7sbvn0wgv7hsnxpss1jba723kx5nz6d4-python-2.7.15/lib/python2.7/logging/handlers.py", line 64, in __init__
one# [    7.767696] graphiteWeb-pre-start[706]:     logging.FileHandler.__init__(self, filename, mode, encoding, delay)
one# [    7.771913] graphiteWeb-pre-start[706]:   File "/nix/store/7sbvn0wgv7hsnxpss1jba723kx5nz6d4-python-2.7.15/lib/python2.7/logging/__init__.py", line 920, in __init__
one# [    7.779395] graphiteWeb-pre-start[706]:     StreamHandler.__init__(self, self._open())
one# [    7.784128] graphiteWeb-pre-start[706]:   File "/nix/store/7sbvn0wgv7hsnxpss1jba723kx5nz6d4-python-2.7.15/lib/python2.7/logging/__init__.py", line 950, in _open
one# [    7.789363] graphiteWeb-pre-start[706]:     stream = open(self.baseFilename, self.mode)
one# [    7.794295] graphiteWeb-pre-start[706]: IOError: [Errno 2] No such file or directory: '/var/db/graphite/log/webapp/info.log'
```